### ISSUE TITLE
bug: videos loop by themselves

### ISSUE THREAD

**Issue by nooseboynoah** [revanced/revanced-patches#321]
### Type

Error at runtime

### Bug description

Videos will loop by themselves. The toggle in the settings doesn't seem to do anything and the same toggle in the video playback option menu doesn't seem to do anything either. This makes the auto play feature for playing the next recommended videos useless. I don't know If this was intentional or it's just a bug. I can't seem to find anyone else having this same issue. Thank you.

### Steps to reproduce

Step 1. Play video
Step 2. Watch the whole video or skip to the end.
Step 3 video replays its self with no way of stopping it or turning it off.

### Relevant log output

```shell
{"cplatform":"mobile","cbr":"app.revanced.android.youtube","cff":"SMALL_FORM_FACTOR","c":"android","cplayer":"ANDROID_EXOPLAYER_V2","cmodel":"SM-G991U","cos":"Android","csdk":"31","cbrver":"17.29.34","cver":"17.29.34","cosver":"12.SP1A.210812.016.G991USQU5CVF8","cbrand":"samsung","videoid":"diHWVptsv9U","cpn":"b4I1LuXr7hDDVNhc","fmt":"244 vp9 854x480@30","afmt":"251 opus","bh":13124,"conn":3,"volume":46,"loudness":"-9.730","bat":"0.900:0","df":"1\/1554","time":"2022-08-08T16:32:57.146Z","glmode":"RECTANGULAR_2D","mtext":"","error":"No errors","logged_in":"1","e":"24253195,24215196,24159074,23964928,24226335,24246428,24143331,24244560,24254628,24084052,24004644,24241231,24241663,24199724,24021124,24222437,24251509,24185614,24228425,24036947,24187377,24232551,39321934,24187043,24164186,39322113,24119845,1714258,24152443,24140247,24114244,24219713,23983296,24123943,24080738,24216068,24245746,24199774,24241936,24078649,24007246,24135310,23925789,23998056,24252544,24227411,23918597,24254837,24244808,24231105,24253408,24220089,24063124,24243112,24177647,24257966,23966208,24161116,23826896,24034168,24077266,24074379,24254110,24168346,24237563,24077241,39322038,24198739,24181174,24211177,24222379,24252017,23946420,24001373,24117491,24246468,24230625,24120820,39321511,24166867,24230811,24252989,24179906,23735348,24252596,24169501,24254566,23888717,24248231,24258218,23744176,24210939,24227054,24191629,24246938,24082169,24247233,24206072,24227844,24248385,24245035,24149832,24253059,39321886,24195035,24235349,24085811,24221292,24224266,23882685,23934970"} 

p.s I don't know if this helps or I can't find the log output so I used the videos debug mode.
```

**Comment by RyuuPendragon** [comment]
I'm also facing this issue (video replays its self repeatedly) on both my mobile and tablet.
ReVanced CLI: 2.9.2
ReVanced Patches: 2.33.2
ReVanced Integrations: 0.31.2
Youtube: 17.29.34

**Comment by masterflitzer** [comment]
might help if you say if this is happening with or without loop patch

**Comment by kazimmt** [comment]
> 

It happens when Loop patch added.

**Comment by ttanathoss** [comment]
Hi, I have the same problem. And it happens also when I'm playing playlist - the app looks like it's loading next video, but plays the same one

**Comment by TheJeterLP** [comment]
Disable auto-repeat in settings.

**Comment by casual-coderr** [comment]
@ttanathoss Go to Settings -> Revanced ->Miscellaneous -> Enable Auto Repeat and turn it off.
Maybe the mods can turn it off by default in future patches.

**Comment by ttanathoss** [comment]
Thanks for the tip. It was disabled but seems like actually enabling it and disabling again fixed the problem

**Comment by Amanse** [comment]
fixed in d43add7
